src/handlers/fedora.ts

```diff
@@ -0,0 +1,133 @@
+import { Alert } from 'react-native';
+import codePush from 'react-native-code-push';
+import {
+  // @ts-ignore
+  APP_CENTER_READ_ONLY_TOKEN_ANDROID,
+  // @ts-ignore
+  APP_CENTER_READ_ONLY_TOKEN_IOS,
+  // @ts-ignore
+  CODE_PUSH_DEPLOYMENT_KEY_ANDROID,
+  // @ts-ignore
+  CODE_PUSH_DEPLOYMENT_KEY_IOS,
+} from 'react-native-dotenv';
+import { rainbowFetch } from '../rainbow-fetch';
+import { Navigation } from '@rainbow-me/navigation';
+import Routes from '@rainbow-me/routes';
+
+const APP_CENTER_READ_ONLY_TOKEN = ios
+  ? APP_CENTER_READ_ONLY_TOKEN_IOS
+  : APP_CENTER_READ_ONLY_TOKEN_ANDROID;
+
+async function checkIfRainbowRelease(deploymentKey: string): Promise<boolean> {
+  try {
+    const response = await rainbowFetch(
+      `https://api.appcenter.ms/v0.1/apps/osdnk/rainbow-${
```

Comment:
this links to your personal `osdnk` ?

Reply:
this needs to be under some account. In appcenter I just create the project, but you (and a bunch of people) are added as well with all permissions so this is just a matter of nice link